src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.h

```diff
@@ -84,6 +88,7 @@ class DefaultDACVerifier : public DeviceAttestationVerifier
 
     CsaCdKeysTrustStore mCdKeysTrustStore;
     const AttestationTrustStore * mAttestationTrustStore;
+    DeviceAttestationRevocationDelegate * mRevocationDelegate = nullptr;
```

Comment:
Suggest having a setter for the revocation delegate.

Reply:
Added the setter API